[15.0][MIG] l10n_es_aeat_mod347

juanpgarza opened the pull request:
Migración standar

Commit:
[REF] *: Refactorización para portar a 6.0 - Paso 1.

- Se han renombrado los módulos para usar la nomenclatura propuesta
  por OpenERP: l10n_es para el módulo base de localización (plan de
  cuentas), l10n_es_* para el resto de módulos.

- Se eliminan los módulos extra_addons/* que deberían moverse a
  los extra-addons genéricos (no son específicos de España).

- Se renombran los __terp__.py por __openerp__.py

Commit:
[REF][REN][FIX] Corregimos errores en los nombres de dependencias de ciertos módulos, por los correspondientes que se modificaron para esta versión 6.0 y en ciertos __init__ adaptamos los imports a los nuevos nombres de los ficheros. Renombrado de los archivos de traducción españoles de es_ES.po a es.po y pequeñas refactorizaciones

Commit:
[FIX+IMP] l10n_es_aeat_mod347: 2 things

* FIX: Renombrado de los archivos de traducción catalanas de ca_ES.po a ca.po
* IMP: Añadidos avisos NO ADAPTADO TODAVÍA A LA VERSIÓN 6.0 a varios módulos, limpieza l10n_es_partner_mercantil

Commit:
[ADD] l10n_es_aeat, l10n_es_aeat_347, l10n_es_aeat_349: Adds a generic module for aeat models, 347 module was portedto v6.0 and adds new module to print AEAT model 349.

Commit:
[IMP+FIX] l10n_es_aeat, l10n_es_aeat_mod347, l10n_es_aeat_mod349: Pequeñas mejoras generales (vistas, traducciones, código) y corrección pequeño bug por un olvido en la adaptación de la v5 a la v6

Commit:
[IMP][FIX] Nos traemos los cambios hechos ayer sobre la versión 5 del módulo del 347, que incluían los cambios hechos por la AEAT para la declaración 2010, para la versión 6.0 del módulo, además corregimos un bug encontrado por Jordi en la versión 5.0 hoy. También añado una nueva comprobación de los registros de empresas, ya que el cif de la empresa es requerido, por lo que no va a dejar confirmar el informe, mientras no se rellene este campo en todos los registros de empresa. La agrupación de pagos en efectivo se dejó igual que en la 5.0, ya que se sigue comportando como en la 5.0, sigo animando a que si alguien conoce como mejorarla adelante.

Commit:
[IMP] l10n_es_aeat_mod347: Actualizada traducción catalana después de las mejoras introducidas en el anterior commit

Commit:
[REF] Cambio de licencia de GPL3 a AGPL en los módulos donde sólo aparecía Pexego en la licencia, quedan los compartidos.

albertca committed:
l10n_es_aeat_mod347: Actualizado el módulo para realizar el nuevo modelo con el detalle de los trimestes obligatorio para la declaración del 2011.

Commit:
[ADD][REF] l10n_ES_aeat_mod347: Añadimos la posibilidad de agrupar por Cif/Nif en el cálculo del 347, por si hay más de una empresa con el mismo cif..., pequeña refactorización usando rec_name en lugar de name_get

Commit:
[FIX] l10n_es_aeat_mod347: Corregimos la agrupación por cif

Commit:
[IMP] l10n_es_aeat_mod347: Actualizadas plantillas y traducciones con los nuevos strings después de los recientes cambios en el 347.

acysos committed:
[FIX] l10n_es_aeat_mod347: Exclusión del IRPF del total de las facturas. Necesario para las Sociedades Irregulares, Sociedades Civiles y Comunidad de Bienes sin regimen de módulos en el IRPF. Al resto de las empresas no les afecta ya que no facturan con IRPF. l10n_es_aeat_mod340: Correción de los autores en el modelo 340

Commit:
Commit inicial de una rama para la versión 7.0. Todos los módulos fueron marcados como no instalables mientras no se compruebe que funcionan o migren, para poder usarlos o probarlos se debería poner el atributo installable de los ficheros __openerp__.py de cada módulo a True

Commit:
Launchpad automatic translations update.

Commit:
[FIX] Fix some typos in pot and po files.

Commit:
Launchpad automatic translations update.

percevaq committed:
[FIX]: Update view definitions to version 7.0.
[FIX]: Add security definition for objects l10n_es_aeat_mod347_calculate_records and l10n_es_aeat_mod347_export_to_boe.
[FIX]: Changed types to orm.Model, orm.TransientModel and orm.AbstractModel.
[FIX]: Contributions have been written in the standard format of the community.
[FIX]: Remove 'init_xml' keys, because it's no longer needed in v7.
[FIX]: Remove __author__ variables in files, because authors are put on manifest file (__openerp__.py).
[FIX]: Increased compatibility with standard PEP8.
[FIX]: Imports in standard format.
[IMP]: Add "state" field in tree view.
[IMP]: Option to generate report suppliers only.
[FIX]: Update address partner search to V7.
[FIX]: Update state in address partner to V7.
[FIX]: Update dependencies, now required the module documents to view generated file in V7.
[FIX]: Translate .pot file to v7.
[FIX]: Translate field "only_supplier" to spanish language.
[FIX]: Format the text that explains the module.
[IMP]: Add icon and screencapture.

Commit:
Launchpad automatic translations update.

Commit:
[IMP] l10n_es_aeat_mod347:
- Cambiado flag de los partners para no incluirlos en el 347 (opción menos común), que incluirlos (que suelen ser todos).
- Eliminado flag para incluir los diarios, ya que no se activa por defecto y el usuario se confunde con los resultados. Se cogen todos los diarios de caja.
- Otras mejoras menores.

Commit:
Launchpad automatic translations update.

Commit:
[IMP] l10n_es_aeat_mod347: Comentarios eliminados y alguna mejora de docstrings.

acysos committed:
[FIX] l10n_es_aeat_mod347: eliminación de las facturas canceladas en el calculo del 347

Commit:
Launchpad automatic translations update.

Commit:
[FIX] l10n_es_aeat_mod347: Corregido cálculo del importe para facturas rectificativas.

Commit:
[IMP] Eliminación de la mayoría de warnings que da runbot.

Commit:
Launchpad automatic translations update.

pedrobaeza committed:
[MIG] Mergear cambios de la 7 y recolocar

pedrobaeza committed:
[MIG] l10n_es_aeat_mod347: Migrado a nueva API

Commit:
[IMP] unificar campos en el módulo base + migrar a la nueva api.

cubells committed:
[IMP] Mejora para poder eliminar facturas selectivamente del modelo 347

pedrobaeza committed:
[FIX+IMP] l10n_es_aeat_mod347: Mejoras en la vista de facturas + Autores/patrocionadores expandidos + PEP8

oca-transbot committed:
OCA Transbot updated translations from Transifex

Commit:
[FIX] l10n_es_aeat_mod347, 340: Fixed terms for 'real estate' in fields, variables, classes and translations, and related PEP8 fixes.

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: copy no existe para las listas

oca-transbot committed:
OCA Transbot updated translations from Transifex

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Compatibilidad API vieja y nueva en la creación de periodos mensuales. Fixes #209

oca-transbot committed:
OCA Transbot updated translations from Transifex

pedrobaeza committed:
[IMP] l10n_es_aeat_mod347: Eliminar la selección de periodo para no confundir

oca-transbot committed:
OCA Transbot updated translations from Transifex

jalzaga committed:
fix code errors in received cash amount

pedrobaeza committed:
[FIX+IMP] l10n_es_aeat_mod347: 3 things:

* FIX: Avoid error when multiple partners with the same VAT
* FIX: Fix onchange_partner_id of partner records
* IMP: Allow to select periods for the calculation

omar7r committed:
[FIX] l10n_es_aeat_mod347: Se corrige una incorrecta selección de empresas sobre las que se calculaba el informe.

oca-transbot committed:
OCA Transbot updated translations from Transifex

angelmoya committed:
WIP MIG mod347

antespi committed:
[8.0][IMP][l10n_es_aeat_mod347] Improve performance and other things (#317)

oca-transbot committed:
OCA Transbot updated translations from Transifex

pedrobaeza committed:
[IMP] l10n_es_aeat_mod347: General code check

oca-travis committed:
[UPD] Update l10n_es_aeat_mod347.pot

mtbochaca committed:
Translated using Weblate (Catalan)

Currently translated at 81.8% (166 of 203 strings)

Translation: l10n-spain-9.0/l10n-spain-9.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-9-0/l10n-spain-9-0-l10n_es_aeat_mod347/ca/

angelmoya committed:
[MIG+IMP] l10n_es_aeat_mod347: Migration to 11.0 + enhancements

pedrobaeza committed:
[FIX+IMP] l10n_es_aeat_mod347: Migration refinements + enhancements

* Add controller for automatic confirmation/rejection
* Improve data model
* Clean code
* Clean UI
* Spanish translation
* Tests
* ...

pedrobaeza committed:
[MIG] l10n_es_aeat_mod347: Migration to 12.0

oca-travis committed:
[UPD] Update l10n_es_aeat_mod347.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_aeat_mod347/

pedrobaeza committed:
[IMP] l10n_es_aeat: Move state mapping to base

oca-travis committed:
[UPD] Update l10n_es_aeat_mod347.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_aeat_mod347/

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Proper variable assignation

So that inherited models are able to access this value.

OCA-git-bot committed:
[UPD] README.rst

pedrobaeza committed:
[IMP] l10n_es_aeat: Add chatter + activities to AEAT models

* Include chatter (mail.thread)
* Include activities (mail.activity.mixin)
* Track AEAT report state change
* Include new admin user in AEAT group by default

oca-travis committed:
[UPD] Update l10n_es_aeat_mod347.pot

OCA-git-bot committed:
l10n_es_aeat_mod347 12.0.1.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_aeat_mod347/

carlosdauden committed:
[FIX] l10n_es_aeat_mod347: report_id not in domain. Error report_347_partner with missed cash fields

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update l10n_es_aeat_mod347.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_aeat_mod347/

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Proper call to action link method

It has been renamed in 12.0 to `_notify_get_action_link`.

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Sign of total amount should be negative if the case

The absolute amount is taken for being compared with the limit, but then the amount
should be consigned with the sign, as stated in:

https://www.agenciatributaria.es/AEAT.internet/Inicio/Ayuda/Modelos__Procedimientos_y_Servicios/Ayuda_Modelo_347/Informacion_general/Preguntas_frecuentes/_Puede_declararse_un_importe_negativo_en_el_modelo_347_.shtml

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Take into account proper sign depending on operation

We can't remove as is the `abs` if we don't take into account possible sign
depending on the operation. Complement of 3927e92699dcec1db26c483d7946ba549c7b8ea8

Included tests for these cases.

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Flake8

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Handle VAT correctly

Since v11, Odoo doesn't need the country code in VATs for performing the checks. You
must fill country + putting the VAT with or without the country code.

This model doesn't work properly for the case where you don't introduce the
country code.

We now use the method created for centralizing all the partner identification stuff
that was introduced with VAT book for performing a proper recognition.

Included tests for checking all of the identification stuff.

oca-travis committed:
[UPD] Update l10n_es_aeat_mod347.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_aeat_mod347/

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Don't treat previous report number as integer

Previously, it was treated as integer, and thus, default for applying sign is applied
as well, resulting in 14 characters instea of 13.

ozono committed:
[FIX] l10n_es_aeat_mod347: real state fixes

Fixed real state onchange bug and some BOE format fields bugs. Tested exporting and uploading sample BOE file to AEAT.

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: More about previous declaration number

2 patches:

* AEAT requires to pass in declaration number a number with 0s if no present, but no
  extra character should be put, so we restore back the integer type, but make sure
  no sign is applied.
* If the declaration is substitutive, we need to put the S in the second character,
  not in the first one.

pedrobaeza committed:
[IMP] l10n_es_aeat_mod347: Don't depend on tax description

Continuation of 611c24bc and later commits. This way, we can decouple tax label from
the matching, thanks to the XML-ID that Odoo gives to created taxes. We also
simplifies the code thanks to this.

oca-travis committed:
[UPD] Update l10n_es_aeat_mod347.pot

OCA-git-bot committed:
l10n_es_aeat_mod347 12.0.1.4.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_aeat_mod347/

pedrobaeza committed:
[IMP] l10n_es_aeat_mod347: Include newly created non deductible taxes in mapping

OCA-git-bot committed:
l10n_es_aeat_mod347 12.0.1.5.0

carlosdauden committed:
[IMP] l10n_es_aeat_mod347: black, isort, prettier

carlosdauden committed:
[MIG] l10n_es_aeat_mod347: Migration to v13

oca-travis committed:
[UPD] Update l10n_es_aeat_mod347.pot

danimv5 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (255 of 255 strings)

Translation: l10n-spain-13.0/l10n-spain-13.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-13-0/l10n-spain-13-0-l10n_es_aeat_mod347/es/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-13.0/l10n-spain-13.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-13-0/l10n-spain-13-0-l10n_es_aeat_mod347/

pedrobaeza committed:
[IMP] l10n_es_aeat_mod347: Add missing non deductible VAT

It was not possible until the merge of odoo/odoo#55491

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-13.0/l10n-spain-13.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-13-0/l10n-spain-13-0-l10n_es_aeat_mod347/

danimv5 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (246 of 246 strings)

Translation: l10n-spain-13.0/l10n-spain-13.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-13-0/l10n-spain-13-0-l10n_es_aeat_mod347/es/

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Adapt odoo/odoo#58062

Now "IVA Exento No Sujeto" is not included in 347, but it should.

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Adapt odoo/odoo#34044

Missing taxes "IVA no sujeto (bienes)" for sales/purchases in the model.

Commit:
[FIX] l10n_es_aeat_mod347: Changing the name of some variables

The fields "X_quarter_real_estate_transmissions_amount"
(X=first, second, third, fourth) had the old names so the report was
failing. The new names for those fields are
"X_quarter_real_estate_transmission".

It was failing in the mail_template as well.

oca-travis committed:
[UPD] Update l10n_es_aeat_mod347.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-13.0/l10n-spain-13.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-13-0/l10n-spain-13-0-l10n_es_aeat_mod347/

jabelchi committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (246 of 246 strings)

Translation: l10n-spain-13.0/l10n-spain-13.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-13-0/l10n-spain-13-0-l10n_es_aeat_mod347/ca/

pedrobaeza committed:
[OU-FIX] l10n_es_aeat_mod347: Missing rename

manuelregidor committed:
[IMP] l10n_es_aeat_mod347: black, isort, prettier

manuelregidor committed:
[WIP][MIG]l10n_es_aeat_mod347: Migration to 14.0

oca-travis committed:
[UPD] Update l10n_es_aeat_mod347.pot

OCA-git-bot committed:
[UPD] README.rst

manuelregidor committed:
[14.0][FIX]l10n_es_aeat_mod_347: correccion readme

oca-travis committed:
[UPD] Update l10n_es_aeat_mod347.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
l10n_es_aeat_mod347 14.0.1.0.1

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_aeat_mod347/

manuelregidor committed:
[14.0][FIX]l10n_es_aeat_mod347: not_in_mod347 checkbox display in invoice view

OCA-git-bot committed:
l10n_es_aeat_mod347 14.0.1.1.0

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Don't include cancel and draft moves

Previously, if there are cancel or draft moves with the corresponding
taxes, they were included in the report.

We should filter them out.

TT33910

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Save proper real state fields

The provided field name was not correct.

Fixes #2001

juanpgarza committed:
[IMP] l10n_es_aeat_mod347: black, isort, prettier

juanpgarza committed:
[MIG] l10n_es_aeat_mod347: Migration to 15.0

HaraldPanten commented:
/ocabot migration l10n_es_aeat_mod347

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Proper real state field

Fixes #2081

pedrobaeza commented:
Haz por favor cherry-pick de 26068b0a3e9b6829fa6b583147aafe69ab11f756

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Proper invoice name

In a customer invoice, the field `name` contains the number of the
invoice, and the field `ref` can contain any other arbitrary content
put by the user, so we should only use this second one in the case of
vendor bills, as this is the invoice number for the vendor.

TT34430

pedrobaeza commented:
Haz por favor cherry-pick de ab392db0039379e13f5dc5324fc8198328539dc3

juanpgarza commented:
> Haz por favor cherry-pick de ab392db0039379e13f5dc5324fc8198328539dc3

sorry Pedro, I can't find this commit...

pedrobaeza commented:
Perdona, no hice push. Ahora sí debe existir: ab392db0039379e13f5dc5324fc8198328539dc3

pedrobaeza commented on l10n_es_aeat_mod347/views/report_347_partner.xml:
```diff
@@ -21,39 +21,149 @@
-                        <div class="col-xs-2">
-                            <strong class="text-right">Amount: <span
-                                    t-field="o.amount"
+                    <t t-if="o.amount">
```
Aquí este diff no parece correcto. Tal vez deberías hacer el cherry-pick antes de la migración y del pre-commit, no después.

pedrobaeza commented on l10n_es_aeat_mod347/controllers/main.py:
```diff
@@ -3,7 +3,7 @@
 from odoo import http
 
-from odoo.addons.mail.controllers.main import MailController
+from odoo.addons.mail.controllers.mail import MailController
```
¿Has comprobado si este controller sigue funcionando bien?

juanpgarza replied:
Si, hice mis pruebas y funcionó correctamente.

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Preserve balance sign on partner move records

When populating move records for a partner, an abs operation is
performed, as AEAT requires to express amount as positive for all
records.

Previous code trusts on the journal entry type for determining later
the sign of the operation, but there can be the possibility of having
a big journal entry where the type is of one sign, but the specific
operation for that partner is another. The canonical example is on PoS,
with the closing entry including together all the operations of the
session.

So let's keep the balance sign all the time, and remove the checks on
move type, converting the amount to positive where needed (BOE export),
and this also makes a bit lighter the quarter computation, not needing
to filter by move_type.

It includes the migration scripts for adapting following the same
previous approach the existing amounts.

TT34561

pedrobaeza commented:
Mientras tanto, ha surgido otro commit que incluir resolviendo un problema: 2102ab1207fb8ed546c4ac9b468c25b02c14f733

pedrobaeza commented:
/ocabot merge nobump

rafaelbn commented:
Hola @juanpgarza , ¿has presentado este mes algún 347 en v15 con esta migración y te ha funcionado?

- [x] No salen en el modelo las facturas de proveedores sólo las de clientes. ¿Fallo local?, al pasar las facturas a borrador y volver a validar ya las coge.
- [x] El importe de las operaciones a veces sale con IVA y a veces sale sin IVA (con el mismo tipo de IVA, 21% (servicios corrientes). ¿Fallo local?, al pasar las facturas a borrador y volver a validar ya las coge.
- [ ] al enviar una línea con el correo electrónico la plantilla sale mal formato, no es incorrecto se podría mejorar.

![imagen](https://user-images.githubusercontent.com/8736623/155735562-adcc46b0-ea7f-4fca-bf28-6bb847d92cba.png)

- [ ] al enviar una línea con el correo electrónico el PDF adjunto "sale" ilegible sin ningún tipo de orden y partidas las tablas a mitad de imagen

![imagen](https://user-images.githubusercontent.com/8736623/155735866-24c63fc7-cdd4-48f1-b57c-7ec5eed527fe.png)

- [x] En las pestañas con vista lista hay un scroll lateral tremendo y no funciona la opción de los tres puntitos para quitar columnas en dicha vista. No se puede reproducir en runboat.


No sé si tienes las mismas incidencias 😄 

Pruebas finales realizadas en http://oca-l10n-spain-15-0-31bff8df6c54.runboat.odoo-community.org/web

Saludos

juanpgarza commented:
> Hola @juanpgarza , ¿has presentado este mes algún 347 en v15 con esta migración y te ha funcionado?
> 
> * [x]  No salen en el modelo las facturas de proveedores sólo las de clientes. ¿Fallo local?, al pasar las facturas a borrador y volver a validar ya las coge.
> * [x]  El importe de las operaciones a veces sale con IVA y a veces sale sin IVA (con el mismo tipo de IVA, 21% (servicios corrientes). ¿Fallo local?, al pasar las facturas a borrador y volver a validar ya las coge.
> * [ ]  al enviar una línea con el correo electrónico la plantilla sale mal formato, no es incorrecto se podría mejorar.
> 
> ![imagen](https://user-images.githubusercontent.com/8736623/155735562-adcc46b0-ea7f-4fca-bf28-6bb847d92cba.png)
> 
> * [ ]  al enviar una línea con el correo electrónico el PDF adjunto "sale" ilegible sin ningún tipo de orden y partidas las tablas a mitad de imagen
> 
> ![imagen](https://user-images.githubusercontent.com/8736623/155735866-24c63fc7-cdd4-48f1-b57c-7ec5eed527fe.png)
> 
> * [x]  En las pestañas con vista lista hay un scroll lateral tremendo y no funciona la opción de los tres puntitos para quitar columnas en dicha vista. No se puede reproducir en runboat.
> 
> No sé si tienes las mismas incidencias smile
> 
> Pruebas finales realizadas en http://oca-l10n-spain-15-0-31bff8df6c54.runboat.odoo-community.org/web
> 
> Saludos

Buenas @rafaelbn. Todavía no me tocó utilizar este módulo en v15. 
Este PR lo hicimos como parte del proyecto de Sesiones mentorizadas de la aeodoo.

Saludos
